pyproject.toml

```diff
@@ -31,7 +32,11 @@ platforms = ["any"]
 
 [tool.setuptools.packages.find]
 where = ["src/"]
-include = ["zinolib", "zinolib.controllers"]
+include = [
+    "zinolib",
+    "zinolib.config",
+    "zinolib.controllers",
+]
```

Comment:
Isn't there a way to have setuptools include all sub-packages of `zinolib` without mentioning them explicitly?

Reply:
I wish! I haven't found it for setuptools without having to resort to setup.py. "flit" can do it. "hatchling" cannot. (I use "flit" for my own projects.)

Reply:
Strange. This is the corresponding section from zino2:

```
[tool.setuptools.packages.find]
where = ["src"]
exclude = ["tests*"]
```

It doesn't use `include` at all, and all subpackages *are* included.

Reply:
I'll try that in some other PR.